.github/actions/setup-project/action.yaml

```diff
@@ -0,0 +1,38 @@
+name: Set up languages and dependencies
+description: Setup python and dependencies and establish database schema
+nputs:
```

Comment:
inputs?

Reply:
No longer need these at all. docker-compose forever and always.